pkg/cmd/scan/output/assets/style.css

```diff
@@ -211,6 +211,7 @@ input[type="search"], select {
 
 .div-right {
     margin: 12px 0;
+    text-align: right;
```

Comment:
Can you change that to `center` and add inside the media query `@media (min-width: 768px)` this line between classes `card` and `panels`:

```css
.div-right {
    text-align: right;
}
```